client/packages/purchase_orders/src/DetailView/DetailView.tsx

```diff
@@ -1,3 +1,12 @@
 import React from 'react';
+import { useParams } from '@openmsupply-client/common';
+import { usePurchaseOrder } from '../api/hooks/usePurchaseOrder';
 
-export const PurchaseOrderDetailView = () => <p>DetailView placeholder</p>;
+export const PurchaseOrderDetailView = () => {
+  const { invoiceId = '' } = useParams();
```

Comment:
carryover - should be PO Id

Reply:
Yeah, good catch, thanks. Have fixed: 58cbcadbd3